flow/ondemand/client/client.go

```diff
@@ -378,6 +378,21 @@ func (o *OnDemandProbeClient) Stop() {
 	o.checkForRegistration.Stop()
 }
 
+// Invoke capture upon start of analyzer on selected from config SubGraph
+func (o *OnDemandProbeClient) InvokeCaptureFromConfig() () {
+	capture_gremlin := config.GetConfig().GetString("analyzer.startup_capture")
+	capture_bpf := config.GetConfig().GetString("analyzer.startup_capture_bpf")
+	if(capture_gremlin == "" && capture_bpf == "") {
```

Comment:
You are missing a space after `if` statement ... generally I would expect `gofmt` to detect and fix stuff like this - you can invoke a check from top-level running `make fmt`

Reply:
ok